### DISCUSSION TITLE
A Simple Thank You

### DISCUSSION THREAD

**Discussion by smartqasa** [punxaphil/custom-sonos-card#176] [General]
This is truly magnificent work. Many thanks.

**Comment by RedDaveNY** [comment]
i add this card and delete all others. Looks clear and nice. 
T H X

**Comment by punxaphil** [comment]
Happy you like my work!

---
PunxaPhil
[![image](https://github.com/user-attachments/assets/cb2e240e-f9ca-4900-a425-b8089216c1ee)](https://patreon.com/PunxaPhil) [![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/9bebc1f6-6bda-4cbd-b596-50d3d2f712de)](https://www.buymeacoffee.com/punxaphil)
